src/content/zh-cn/fundamentals/getting-started/codelabs/your-first-pwapp/index.md

```diff
@@ -150,82 +143,55 @@ Note: 在寻找更多信息? 观看 Alex Russell 在 2015 年 Chrome 开发者
 * 一个用来添加城市的对话框
 * 一个加载指示器
 
-
-
-    <!DOCTYPE html>
-    <html>
-    <head>
-      <meta charset="utf-8">
-      <meta http-equiv="X-UA-Compatible" content="IE=edge">
-      <meta name="viewport" content="width=device-width, initial-scale=1.0">
-      <title>Weather App</title>
-      <!-- Insert link to styles.css here -->
-    </head>
-    <body>
-      <header class="header">
-        <h1 class="header__title">Weather App</h1>
-        <button id="butRefresh" class="headerButton"></button>
-        <button id="butAdd" class="headerButton"></button>
-      </header>
-    
-      <main class="main" hidden>
-        <!-- Insert forecast-card.html here -->
-      </main>
-    
-      <div class="dialog-container">
-        <!-- Insert add-new-city-dialog.html here -->
-      </div>
-    
-      <div class="loader">
-        <svg viewBox="0 0 32 32" width="32" height="32">
-          <circle id="spinner" cx="16" cy="16" r="14" fill="none"></circle>
-        </svg>
-      </div>
-    
-      <!-- Insert link to app.js here -->
-    </body>
-    </html>
-    
-
-需要注意的是默认情况下 `main` 是隐藏（`hidden`） 的，而加载指示器是显示出来的。这是为了保证
+```
```

Comment:
To get the proper code highlighting, code should be indented 4 spaces. Wrapping in ``` unfortunately doesn't work